test/main/signers/trezor/bridge.test.js

```diff
@@ -0,0 +1,165 @@
+import TrezorConnect, { DEVICE_EVENT, DEVICE, UI_EVENT, UI } from 'trezor-connect'
+import { EventEmitter } from 'stream'
+import log from 'electron-log'
+
+import TrezorBridge from '../../../../main/signers/trezor/bridge'
+
+jest.mock('trezor-connect')
+
+const events = new EventEmitter()
+
+beforeAll(() => {
+  log.transports.console.level = false
+
+  TrezorConnect.on = events.on.bind(events)
+  TrezorConnect.once = events.once.bind(events)
+  TrezorConnect.emit = events.emit.bind(events)
+  TrezorConnect.removeAllListeners = events.removeAllListeners.bind(events)
+})
+
+afterAll(() => {
+  log.transports.console.level = 'debug'
+})
+
+beforeEach(done => {
+  TrezorBridge.once('connect', done)
+  TrezorBridge.open()
+})
+
+afterEach(async () => TrezorBridge.close())
```

Comment:
Not sure you need the async, not sure you need to return the result of `close()` either

Reply:
the reason for this is that the hook will wait for the promise to resolve before completing

Reply:
https://jestjs.io/docs/setup-teardown#repeating-setup

Reply:
oh whoops forgot `close()` wasn't async, you're right, will fix